### PR TITLE
Enable whats-new for 1.52 in stable

### PR DESCRIPTION
**Pull request by simonhong**

### CODE REVIEW

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.52.115 Chromium: 114.0.5735.53 (Official Build) (x86_64)
-- | --
Revision | c499d7ea22c8b2dba278465a5df7b86a8efa4e64-refs/branch-heads/5735@{#970}
OS | macOS Version 11.7.7 (Build 20G1345)

### Steps:
1. installed `1.52.115`
2. launched Brave using `--variations-server-url=https://variations.bravesoftware.com/seed --update-feed-url=https://updates.bravesoftware.com/sparkle/Brave-Browser/test/appcast.xml`
3. checked for updates
4. waited for the update to be downloaded and installed
5. clicked on `Relaunch`
6. upon relaunch, examined tabs

### Confirmed the `What's new in Brave?` page opened in a new-tab window to the left of the previous tabs


example | example | example | example
---------|---------|----------|---------
Screen Shot 2023-05-30 at 12 51 06 PM | Screen Shot 2023-05-30 at 12 52 22 PM | Screen Shot 2023-05-30 at 12 52 31 PM | Screen Shot 2023-05-30 at 12 52 35 PM

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.52.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
